### PR TITLE
Fix deadblog colours for DCR

### PR DESCRIPTION
**Pull request by ioannakok**
## What does this change?
Makes DCR deadblog colours the same as AR as per this PR: https://github.com/guardian/dotcom-rendering/pull/3823

## Why?
To match the designs and follow accessibility standards.

Background | White | Gray
-- | -- | --
News | 400 | 400
Sport | 400 | 300
Lifestyle | 400 | 300
Culture | 350 | 300
Opinion | 300 | 200 for dark gray and 300 for light gray

| Before      | After |
| ----------- | ----------- |
| image      | image       |
| image   | image  |
| image | image |
| image | image  |
| image | image |
| image | image |
| image | image |
| image | image |
| image | image |
| image | image |

### CODE REVIEW

File: dotcom-rendering/src/web/lib/decidePalette.ts [hunk 1]
```diff
@@ -731,7 +817,22 @@ const hoverHeadlineByline = (format: ArticleFormat): string => {
 
 const textRichLink: (format: ArticleFormat) => string = (format) => {
 	if (format) {
-		return pillarPalette[format.theme].main;
+		switch (format.theme) {
```
Comment (DavidLawes) [inline dotcom-rendering/src/web/lib/decidePalette.ts:820]: These changes aren't scoped only to deadblogs (?) - is this ok?

File: dotcom-rendering/src/web/lib/decidePalette.ts [hunk 2]
```diff
@@ -204,6 +202,25 @@ const textSyndicationButton = (format: ArticleFormat): string => {
 };
 
 const textArticleLink = (format: ArticleFormat): string => {
+	if (format.design === ArticleDesign.DeadBlog) {
```
Comment (DavidLawes) [inline dotcom-rendering/src/web/lib/decidePalette.ts:205]: I like how you've separated out the requirements for deadblogs, I can see how we could refactor (to reduce lines of code) but I prefer your approach :)

**Review by DavidLawes — APPROVED** [review]
Thanks for the PR description - the changes look very detailed and specific! Out of curiosity, would we expect any chromatic diffs as a result of this?

**Comment by ioannakok** [comment]
> Thanks for the PR description - the changes look very detailed and specific! Out of curiosity, would we expect any chromatic diffs as a result of this?

Yes! You are right. Here are the diffs I have accepted:
https://www.chromatic.com/build?appId=5dfcbf3012392c0020e7140b&number=9844
https://www.chromatic.com/build?appId=5dfcbf3012392c0020e7140b&number=9846

I did this incrementally, so they don't show in the final version of the PR. Should have accepted them at the end.

**Review by mxdvl — APPROVED** [review]
This looks good to me… quite easy to get my head around!

File: dotcom-rendering/src/web/lib/decidePalette.ts (hunk 1 above)
Comment (ioannakok) [reply dotcom-rendering/src/web/lib/decidePalette.ts:820]: In RichLink we pass the format of the article we're linking to, e.g. when we're linking to an article, the design will be Standard and not Deadblog. So if we add a case for deadblogs, it won't be reached. I was ambivalent as to what to do but decided to make the change as Culture and Lifestyle richLinks are not accessible atm. Thanks for pointing this out, I will double check with @HarryFischer!

Comment (ioannakok) [reply dotcom-rendering/src/web/lib/decidePalette.ts:820]: It is okay to make this change 👍

**Pull request merged**